src/main/java/core/basesyntax/strategy/handler/BalanceOperationHandler.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax.strategy.handler;
+
+import core.basesyntax.dao.FruitDao;
+import core.basesyntax.dao.FruitDaoImp;
+import core.basesyntax.model.FruitTransaction;
+
+public class BalanceOperationHandler implements OperationHandler {
+    private FruitDao dataBase;
+
+    public BalanceOperationHandler() {
+        this.dataBase = new FruitDaoImp();
+    }
+
+    @Override
+    public boolean doOperation(FruitTransaction fruit) {
```

Comment:
1) handler should handle
2) We're inside of **operation**Handler, why does it accept fruit of type FruitTransaction?